tsconfig.json

```diff
@@ -68,5 +68,8 @@
     "./scripts/*",
     "./documentation/samples",
     "./documentation/scripts"
-  ]
+  ],
+  "ts-node": {
```

Comment:
I wasn't able to run `mocha test/upload_validation_test.ts` without this locally.

Reply:
Ah ok, yeah we just use `make test-file` normally.